web/src/pages/Cases/CaseDetails/Overview/DisputeBasicInfo.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import ReactMarkdown from "react-markdown";
```

Comment:
let's import `ReactMarkdown` from `components/ReactMarkdown`, we had some instances where the description was too big, in `components/ReactMarkdown` we set font size to 16px always to keep it looking more within the style

Reply:
Done ✅